doc/source/dev/reviewer_guidelines.rst

```diff
@@ -28,21 +57,33 @@ Reviewer Checklist
 - Does the documentation follow the :ref:`NumPy guidelines<howto-document>`? Are
   the docstrings properly formatted?
 - If it is not obvious from the PR description, add a short explanation of what
-  a branch did to the merge message and, if closing a bug, also add "Closes
+  a branch did to the merge message and, if closing an issue, also add "Closes
   #123" where 123 is the issue number.
+- For code changes, at least one core team member (those with commit rights)
+  should review and approve all pull requests. If you are the first to review a
```

Comment:
Not sure of the value of the first sentence. The only way the PR can be committed to master is if someone with commit rights does it.
```suggestion
-  If you are the first to review a
```